api/cpp/docs/index.rst

```diff
@@ -35,6 +35,15 @@ Slint C++ documentation
 
    genindex
 
+.. toctree::
+   :hidden:
+   :maxdepth: 2
+   :caption: Advanced Topics
+
+   Debugging Techniques <../slint/src/advanced/debugging_techniques.html#http://>
+   Selecting a Widget Style <../slint/src/advanced/style.html#http://>
+   Platform Backends <../slint/src/advanced/backends.html#http://>
```

Comment:
Can we have comments in .rst  and add a comment mentioning https://github.com/sphinx-doc/sphinx/issues/701#issuecomment-697116337 ?

Reply:
Yes, good point. If we proceed with this, then I'll add that.